### ISSUE TITLE
Update JSON-Schema to version Draft-07

### ISSUE THREAD

**Issue by korzio** [korzio/djv#60]
http://json-schema.org/documentation.html

**Comment by jacobbogers** [comment]
any progress on this? what time and effort would it take? maybe can help out

**Comment by korzio** [comment]
@jacobbogers Unfortunately, didn't work on it yet. I don't expect it is a really tough, but it can be a couple of days work.
Sure, I would appreciate any contribution, also can help with reviews and hints. Do you want to work on it? Would be great!

**Comment by jacobbogers** [comment]
ok cool, so it implements draft 6 fully right, ? let me take a looksi, found your project via the benchmark page, you are the fastest ,   but not support draft 7,  If i do it will queue it for next weekend 👷 (contribute  other opens source projects)

**Comment by korzio** [comment]
Right, the 6th draft is implemented. Good luck and let me know if you need any help!

**Comment by Relequestual** [comment]
*ahem* draft-8 is now impending.

**Comment by Bessonov** [comment]
16 September 2019: Draft 2019-09 (formerly known as draft-08) has been published! :)

**Comment by handrews** [comment]
@korzio draft-07 is a fairly straightforward superset of draft-06

draft 2019-09 has more substantial changes, but we are hoping that it will be the stable base for finalizing the standard.  There are a few of the new areas where we still need some feedback to finish things up.  Note that OpenAPI 3.1 will use JSON Schema draft 2019-09, which will remove one of the main reasons people have stayed on older drafts.

**Comment by korzio** [comment]
Thanks @handrews , hopefully we'll be able to implement new standard versions soon

**Comment by philsturgeon** [comment]
@korzio hey, I didnt know you were in Amsterdam or I'd have come said hello. I'm in the market for a new JavaScript JSON Schema validator, and DJV could well be it. If you can get 2019-09 support in there then absolutely. Do you have an OpenCollective we can support?

**Comment by korzio** [comment]
Hey @philsturgeon Thanks! Next time you are in Amsterdam - let me know 😄No, don't have OpenCollective yet, would you suggest to start it?

**Comment by xXMarcinatorXx** [comment]
Any idea of when draft-08 is supported?

**Comment by philsturgeon** [comment]
@korzio I'll be there next week after API Days Paris, we can chat over a beverage.

**Comment by handrews** [comment]
@korzio @Relequestual just set up an OpenCollective for the JSON Schema project so he might have thoughts on that.

**Comment by korzio** [comment]
@handrews thanks!
We have chatted with @philsturgeon openAPI
If you guys @Relequestual have any proposals anyways, would be glad to hear.
So far draft7 is in plans, but not in wip unfortunately.

**Comment by Relequestual** [comment]
@korzio Happy to chat about this if you'd like! Please find / DM me on the JSON Schema slack =]
